examples/classic-typescript/docs/tutorial-extras/translate-your-site.md

```diff
@@ -43,7 +43,7 @@ Your localized site is accessible at [http://localhost:3000/fr/](http://localhos
 
 :::caution
 
-In development, you can only use one locale at a same time.
+In development, you can only use one locale at the same time.
```

Comment:
Thanks

Examples shouldn't be updated, they are generated from our init templates

The fix should be done here instead: packages/create-docusaurus/templates/shared/docs/tutorial-extras/translate-your-site.md

Reply:
Updated the `create-docusaurus` package. Should the commit changing examples be reverted?